src/components/Tos/DropdownInput/DropdownInput.js

```diff
@@ -81,28 +96,39 @@ export const DropdownInput = ({
         });
       }
     }
+    const valueArray = valueState instanceof Array ? valueState : [valueState];
+    forEach(valueArray, function(value) {
+      if (!find(optionsArray, function(option) { return option.value === value; })) {
```

Comment:
These should be abstracted away into function that's named for what the check is about.

I don't quite get what is happening here, but having names for checks would help.